crates/stellar-xdr-wasm/src/lib.rs

```diff
@@ -0,0 +1,96 @@
+mod utils;
+
+use std::{
+    cmp,
+    io::{self, Read},
+    str::FromStr,
+};
+
+use stellar_xdr::{
+    curr::{self, TypeVariant, WriteXdr},
+    schema::Schema,
+};
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub fn type_variants() -> String {
+    format!(
+        "[{}]",
+        TypeVariant::VARIANTS_STR
+            .iter()
+            .map(|s| format!("{s:?}"))
+            .collect::<Vec<String>>()
+            .join(", ")
```

Comment:
We should be able to return a `JsValue` and use the `Array` type to build an array of strings to return to the JS side. Is that compatible with what we're doing here?